From b5065307ad3480ec21c82169ed66a9e05898b908 Mon Sep 17 00:00:00 2001
From: Jehan <jehan@girinstud.io>
Date: Tue, 12 Jun 2018 14:26:31 +0200
Subject: [PATCH] desktop: rename launchable in appdata.

Our desktop is indeed gimp.desktop, but this is a test to check a
potential bug on flatpak side.
---
 desktop/org.gimp.GIMP.appdata.xml.in.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/desktop/org.gimp.GIMP.appdata.xml.in.in b/desktop/org.gimp.GIMP.appdata.xml.in.in
index 57c47a0084..ac56eabcdb 100644
--- a/desktop/org.gimp.GIMP.appdata.xml.in.in
+++ b/desktop/org.gimp.GIMP.appdata.xml.in.in
@@ -63,7 +63,7 @@
   <project_group>GNOME</project_group>
   <translation type="gettext">gimp20</translation>
 
-  <launchable type="desktop-id">gimp.desktop</launchable>
+  <launchable type="desktop-id">org.gimp.GIMP.desktop</launchable>
   <provides>
     <binary>@GIMP_COMMAND@</binary>
     <!-- Fix the ID renaming, and things like past reviews made on the
-- 
2.17.1

